packages/terra-card/tests/jest/Card.test.jsx

```diff
@@ -14,6 +14,13 @@ describe('Card', () => {
   it('should use the default value when no value is given', () => {
     const wrapper = shallow(defaultRender);
     expect(wrapper.find('.card'));
+    expect(wrapper.find('.default'));
+  });
+
+  it('should use the raised variant when prop is provided', () => {
+    const wrapper = shallow(<Card variant="raised" />);
+    expect(wrapper.find('.raised'));
```

Comment:
Do we need the wrapper.find tests? Your snapshot had a card with the raised class on it. To me, it feels like duplicate testing when we do test jest like this.

Reply:
My personal opinion is that snapshot tests are not very helpful and often times ignored. I typically prefer explicit jest tests. 
That being said, i can remove the test if you want